packages/router/src/grpc/view-protocol-server/balances.ts

```diff
@@ -109,6 +120,39 @@ class BalancesAggregator {
     );
     currentAmount.lo = newAmount.lo;
     currentAmount.hi = newAmount.hi;
+
+    await this.aggregateEquivalentValues(valueView, toAdd);
```

Comment:
How about we pull this out of this function so it sits on the parent alongside:

```typescript
await this.aggregateAmount(valueView, n);
await this.aggregateEquivalentValues(valueView, toAdd);
```

Reply:
Good call, done